scala/private/rule_impls.bzl

```diff
@@ -561,29 +644,62 @@ def _lib(ctx, base_classpath, non_macro_lib):
       jars_to_labels = jars.jars2labels,
     )
 
+def get_unused_dependency_checker_mode(ctx):
+  if ctx.attr.unused_dependency_checker_mode:
```

Comment:
isn't this always set since you have a default?

Reply:
👍 left from when I didn't have the default

Reply:
yep. got it.